build.xml

```diff
@@ -283,6 +282,7 @@ message.
 		</parallel>
 
 		<ant dir="portal-impl" inheritAll="false" target="compile" />
+		<ant dir="portal-test" inheritAll="false" target="compile" />
```

Comment:
@dantewang @ChiZelin 
Why we need to do this? Did we import portal-impl into portal-test-integration before?

Tina

Reply:
@tinatian Yes, we did. For example, please see MailServiceTestUtil. It imports com.liferay.portal.util.PrefsPropsUtil.
cc @dantewang

Reply:
Hi @dantewang @ChiZelin,
How many dependencies we have?  Let's try to remove the dependency on portal-impl first, then move to portal-test.

For example PrefsPropsUtil is just add a layer of caching which is not important to test, we can use _portalPreferencesLocalService directly.

I am going to close this pull, please resend

Tina.

Reply:
Hi @tinatian 
There are dependencies on portal-impl:

Classes | Dependencies
-- | --
PermissionCheckerMethodTestRule | com.liferay.portal.security.permission.SimplePermissionChecker
TikaSafeRandomizerBumper | com.liferay.portal.util.FileImpl
MailServiceTestUtil | com.liferay.portal.util.PrefsPropsUtil
ServiceTestUtil | com.liferay.portal.util.PortalInstances

Some of them seem hard to be removed, especially ```PortalInstances```.
And AspectJNewEnvTestRule has dependency ```com.liferay.util.SerializableUtil```  in ```util-java```, but it is easy to remove it.

What should we do next?
cc @dantewang